lighthouse-core/gather/gatherers/dobetterweb/js-vulnerable-libraries.js

```diff
@@ -0,0 +1,100 @@
+
+  let libraries = [];
+  for (let i in d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests) {
+    try {
+      const result = d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests[i].test(window);
+      if (result === false) continue;
+      libraries.push({
+          name: i,
+          version: result.version,
+          npmPkgName: d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests[i].npm
+      });
+    } catch(e) {
+      console.log('Library Detector test for ' + i + ' failed:', e);
```

Comment:
Removed. Doesn't look like LH typically records things like this and it's unused anyway.